hypothesis/via
# Enable caching for proxied content

To test this:

 * `make docker`
 * `docker stop via3_nginx-proxy_1; docker rm via3_nginx-proxy_1; make services`
 * Now you should be able to follow the following URLs with the following outcomes:
 * `curl -I http://localhost:9083/proxy/static/http://pdf995.com/samples/pdf.pdf`
   * Upstream has Cache-Control
   * `Cache-Control: public, max-age=86400, stale-while-revalidate=604800`
 * `curl -I http://localhost:9083/proxy/static/http://africau.edu/images/default/sample.pdf`
   * Upstream has no Cache-Control
   * `Cache-Control: public, max-age=86400, stale-while-revalidate=604800`
 * `curl -I http://localhost:9083/proxy/static/http://httpbin.org/status/500`
   * No cache header
 * `curl -I http://localhost:9083/proxy/static/http://httpbin.org/status/500`
   * No cache header

----

Net effect:

 * Previously Cloudflare defaults kicked in (now 30m)
 * Now we set to 1 day
 * We now set stale while revalidate
